lib/livereload.js

```diff
@@ -73,8 +73,12 @@ function StartServer() {
             }
 
             // After a file changes, first run prepare, then reload.
-            Q().then(function() {
-                self.context.cordova.prepare();
+            return self.context.cordova.raw.prepare().then(function() {
+                // Remove CSP directive from HTML
+                var CSPRemover = require('./utils/CSPRemover');
```

Comment:
Nit: consider defining CSPRemover further up as on every change you do not need to run 'require'